tests/test_conflict_event.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+
+from hdx.database import Database
+
+from hapi_schema.db_conflict_event import (
+    DBConflictEvent,
+    view_params_conflict_event,
+)
+
+
+def test_conflict_event_view(run_view_test):
+    """Check that national risk references other tables."""
+    view_conflict_event = Database.prepare_view(
+        view_params_conflict_event.__dict__
+    )
+    run_view_test(
+        view=view_conflict_event,
+        whereclause=(
+            view_conflict_event.c.resource_hdx_id
+            == "90deb235-1bf5-4bae-b231-3393222c2d01",
+            view_conflict_event.c.location_name == "Foolandia",
+            view_conflict_event.c.admin1_name == "Province 01",
+            view_conflict_event.c.admin2_name == "District A",
+        ),
+    )
+
+
+def _sample_data():
```

Comment:
If you make this a pytest fixture then you can just pass it to each test directly and avoid calling it every time

Reply:
I want to call it every time so that I can make a new dict, and then the test can change just what it needs to change. That way, there's less to fill in.